js/index.js

```diff
@@ -1 +1,16 @@
-// JS goes here
\ No newline at end of file
+// JS goes here
+
+const navOpen = document.querySelector(".open-btn");
+const navClose = document.querySelector(".close-btn");
+const bigNav = document.querySelector(".nav-big");
+const smallNav = document.querySelector(".nav-small");
+
```

Comment:
`querySelectors`correctly used with class names to create objects that then call `addEventListner` methods of `click` events.